src/Parser/Hunter/BeastMastery/Modules/Spells/DireBeast/DireBeast.js

```diff
@@ -0,0 +1,122 @@
+        const reductionMs = this.spellUsable.reduceCooldown(SPELLS.BESTIAL_WRATH.id, COOLDOWN_REDUCTION_MS);
+        if (this.remainingBestialWrathCooldown > CD_ON_BESTIAL_WRATH_BAD_DB_THRESHHOLD) {
+          this.effectiveBWReduction += reductionMs;
+          this.wastedBWReduction += (COOLDOWN_REDUCTION_MS - reductionMs);
+        } else {
+          this.badDBCasts += 1;
+          this.effectiveBWReduction += reductionMs;
+          this.wastedBWReduction += (COOLDOWN_REDUCTION_MS - reductionMs);
+        }
+      } else {
+        this.wastedBWReduction += COOLDOWN_REDUCTION_MS;
+        this.badDBCasts += 1;
+      }
+      this.DireBeastCastsSinceLastBW += 1;
+    }
+    if (spellId === SPELLS.BESTIAL_WRATH.id) {
+      this.bestialWrathCast = event.timestamp;
+      this.DireBeastCastsSinceLastBW = 0;
+    }
+  }
+
+  statistic() {
+    const gainedBestialWraths = this.effectiveBWReduction / BESTIAL_WRATH_BASE_CD;
+    return (
+      <StatisticBox
+        icon={<SpellIcon id={SPELLS.DIRE_BEAST.id} />}
+        value={(
+          <span>
+            {`gained `}{formatNumber(gainedBestialWraths, 1)}{'  '}
+            <SpellIcon
+              id={SPELLS.BESTIAL_WRATH.id}
+              style={{
+                height: '1.3em',
+                marginTop: '-.1em',
+              }}
+            />
+            <br />
+            {this.casts}{'  '}
+            <SpellIcon
+              id={SPELLS.DIRE_BEAST.id}
+              style={{
+                height: '1.3em',
+                marginTop: '-.1em',
+              }}
+            />
+            {'  '}
+            {this.badDBCasts}{'  '}
+            <SpellIcon
+              id={SPELLS.DIRE_BEAST.id}
+              style={{
+                height: '1.3em',
+                marginTop: '-.1em',
+                filter: 'grayscale(100%)',
+              }}
+            />
+          </span>
+        )}
+        label={`Direbeast information`}
+        tooltip={`You cast Dire Beast ${this.casts} times. <br/> <ul> <li>This reduced Bestial Wraths cooldown by ${(this.effectiveBWReduction / 1000).toFixed(1)} seconds in total, which resulted in you gaining ${formatNumber(gainedBestialWraths, 2)} extra Bestial Wrath casts. </li> <li>You lost out on ${this.wastedBWReduction / 1000} seconds of CD reduction by casting Dire Beast while Bestial Wrath wasn't on cooldown or while cooldown had less than ${COOLDOWN_REDUCTION_MS / 1000} seconds remaining. </li><li> You cast ${this.badDBCasts} Dire Beasts while there was less than 3 seconds remaining of Bestial Wrath cooldown.</li></ul>`}
+      />
+    );
+  }
+  suggestions(when) {
+    when(this.badDBCasts).isGreaterThan(0)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span>Delay casting <SpellLink id={SPELLS.DIRE_BEAST.id} /> if there is less than 3 seconds cooldown remaining on <SpellLink id={SPELLS.BESTIAL_WRATH.id} />.</span>)
```

Comment:
It's not clear to me from reading relevant tooltips *why* it's so bad to cast Dire Beast with 3 seconds or less remaining on Bestial Wrath. If it's not clear to me it's probably also not clear to a newbie BM player looking for advice in suggestions. Perhaps give some more explanation.

Reply:
That makes sense, will add!